espmonitor/src/lib.rs

```diff
@@ -348,6 +350,7 @@ fn process_line(reader: &SerialReader, line: &str) -> String {
                 .ok()
                 .and_then(|output| String::from_utf8(output.stdout).ok())
             {
+                println!("output: {}", output);
```

Comment:
I assume this is some leftover debugging and should be removed?

Reply:
yes